### PR TITLE
Add Scylla Cloud base longevity pipelines

### PR DESCRIPTION
**Pull request by fruch**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (`hydra unit-tests`)
- ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by bentsi** [comment]
is it ready for review?

File: sdcm/cluster.py [hunk 1]
```diff
@@ -165,6 +165,7 @@ class Setup(object):
     MIXED_CLUSTER = False
     MULTI_REGION = False
     BACKTRACE_DECODING = False
+    INTERNODE_COMM_PUBLIC = False
```
Comment (bentsi) [inline sdcm/cluster.py]: INTRANODE

File: vars/scyllaCloudLongevityPipeline.groovy [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            string(defaultValue: "${pipelineParams.get('n_db_nodes', '3')}",
+                   description: 'any type support by scylla cloud',
+                   name: 'n_db_nodes')
+
+            string(defaultValue: "${pipelineParams.get('provision_type', 'spot_low_price')}",
+                   description: 'spot_low_price|on_demand|spot_fleet|spot_low_price|spot_duration',
+                   name: 'provision_type')
+
+            string(defaultValue: "${pipelineParams.get('post_behaviour', 'destroy')}",
+                   description: 'keep|destroy',
```
Comment (bentsi) [inline vars/scyllaCloudLongevityPipeline.groovy]: add a comment: for loader nodes

File: vars/scyllaCloudLongevityPipeline.groovy [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                            echo "start test ......."
+                            ./docker/env/hydra.sh run-test ${pipelineParams.test_name} --backend aws-siren --logdir /sct
+                            echo "end test ....."
+                           """
+                        }
+                    }
+                }
+            }
+        }
+        post {
```
Comment (bentsi) [inline vars/scyllaCloudLongevityPipeline.groovy]: add ability to skip this step in order to keep cluster alive to debug issues

File: sdcm/cluster.py (hunk 1 above)
Comment (fruch) [reply sdcm/cluster.py]: done

File: vars/scyllaCloudLongevityPipeline.groovy (hunk 2 above)
Comment (fruch) [reply vars/scyllaCloudLongevityPipeline.groovy]: no needed, now that this param is also used to destroy the scylla could

File: vars/scyllaCloudLongevityPipeline.groovy (hunk 3 above)
Comment (fruch) [reply vars/scyllaCloudLongevityPipeline.groovy]: Added

**Comment by fruch** [comment]
A 3h job running:
https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/empty/job/longevity/job/scylla-cloud-longevity-10gb-3h/14

**Pull request merged**